docs/orleans/overview.md

```diff
@@ -1,190 +1,133 @@
 ---
 title: Orleans overview
 description: An introduction to .NET Orleans.
-ms.date: 03/10/2022
+ms.date: 05/10/2022
 ---
 
 # Microsoft Orleans
 
-Orleans is a cross-platform framework for building robust, scalable distributed applications. It builds on the developer productivity of .NET and brings it to the world of distributed applications, such as cloud services. Orleans scales from a single on-premises server to globally distributed, highly-available applications in the cloud.
+Orleans is a cross-platform framework for building robust, scalable distributed applications. Distributed applications are defined as apps that span more than a single process, often beyond hardware boundaries using peer-to-peer communication. Orleans scales from a single on-premises server to globally distributed, highly-available applications in the cloud. Orleans extends familiar concepts and C# idioms to multi-server environments. Orleans is designed to scale elastically. When a host joins a cluster it can accept new activations. When a host leaves the cluster, either because of scale down or a machine failure, the previous activations on that host will be re-activated on the remaining hosts as needed. An Orleans cluster can be scaled down to a single host. The same properties which enable elastic scalability also enable fault tolerance. The cluster automatically detects and quickly recovers from failures.
```

Comment:
```suggestion
Orleans is a cross-platform framework for building distributed, cloud-native applications that abstract away the hardware and enable you to scale from a single on-premises server to globally-distributed, highly-available applications. Orleans extends familiar concepts and C# idioms to multi-server environments. Orleans is designed to scale elastically. When a host joins a cluster it can accept new activations. When a host leaves the cluster, either because of scale down or a machine failure, the previous activations on that host will be re-activated on the remaining hosts as needed. An Orleans cluster can be scaled down to a single host. The same properties which enable elastic scalability also enable fault tolerance. The cluster automatically detects and quickly recovers from failures.
```